components/authz-service/storage/postgres/migration/sql/49_add_status_field_to_rules.up.sql

```diff
@@ -0,0 +1,154 @@
+BEGIN;
```

Comment:
ℹ️ Updated SQL functions to return the "status" = "applied"/"staged".

Converted to use `json_build_object` in the process.